x/committee/types/codec.go

```diff
@@ -93,6 +93,8 @@ func RegisterInterfaces(registry types.InterfaceRegistry) {
 		(*PubProposal)(nil),
 		&Proposal{},
 		&govtypes.TextProposal{},
+		&upgradetypes.SoftwareUpgradeProposal{},
```

Comment:
also added this since I was testing it out with a proposal file

Reply:
nice, I believe we have all the proposals covered.  I'm curious if the Committee change and delete proposals are required here